docs/source/_templates/theme_variables.jinja

```diff
@@ -4,12 +4,12 @@
   'contributing': 'https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md',
   'docs': 'https://pytorch.org/ignite/index.html',
 
-  'home': 'https://pytorch.org/ignite',
+  'home': 'https://pytorch-ignite.ai',
```

Comment:
Let's keep home as it was such that clicking on the logo user stays on the same page. And can go to the main site by clicking on "pytorch-ignite.ai"